kolibri/content/utils/paths.py

```diff
@@ -16,6 +16,8 @@
 # set of file extensions that should be considered zip files and allow access to internal files
 POSSIBLE_ZIPPED_FILE_EXTENSIONS = set([".perseus", ".zip", ".epub", ".epub3"])
 
+def get_content_file_name(obj):
+    return '{checksum}.{extension}'.format(checksum=obj.id, extension=obj.extension)
```

Comment:
Yeah, just hard to figure out *where* we would ban it. :) But hopefully it's edge-casey enough, and not worth trying to do intentionally, that it won't turn up in practice. :crossed_fingers: